swagger.yaml

```diff
@@ -4463,7 +4459,11 @@ definitions:
       schedulable_masters:
         type: boolean
         description: Schedule workloads on masters
-        default: false
+        default: true
+      use_scheduling_defaults:
+        type: boolean
+        description: False if the scheduling of workloads on masters has been set by the user through the API.
```

Comment:
When would a user set this?

Based on the description it sounds like this value is not something that is provided by the user, but then why is it in the update API?

Reply:
The user must set `use_scheduling_defaults` to `True` to return to the expected defaults which are calculated depending on the number of the hosts in the cluster.

Reply:
This comment doesn't make sense to me.
How can something be a "default" if it's "calculated depending on the number of hosts"?

Reply:
With "expected defaults" I mean the expected values which are set by the _openshift-install_ command depending on the number of hosts. For less than 5 hosts or more the default expected value would be `true` and `false` otherwise.